packages/core/src/controllers/ConvertController.ts

```diff
@@ -385,12 +385,15 @@ export const ConvertController = {
       return
     }
 
-    const value = res.instant
+    const value = res.standard
     const gasFee = BigInt(value)
     const gasLimit = BigInt(INITIAL_GAS_LIMIT)
     const gasPrice = this.calculateGasPriceInUSD(gasLimit, gasFee)
 
     state.gasPriceInUSD = gasPrice
+
+    // eslint-disable-next-line consistent-return
```

Comment:
I guess rather than doing an empty return on top, we could handle this better way to give better feedback to the user. When the endpoint return an error, what would happen on the UI side?

I could tackle this in the improvements of the convert.

Reply:
Instead of disabling the rule, wouldn't it make more sense to return something like 
`{ gasPrice: null, gasPriceInUsd: null }` on the earlt return?